test/support/fake_slack.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Slack.FakeSlack do
```

Comment:
This could/should(?) be moved out into the public API to help others test their apps.